typing/env.ml

```diff
@@ -824,14 +827,41 @@ let find proj1 proj2 path env =
 
 let find_value =
   find (fun env -> env.values) (fun sc -> sc.comp_values)
-and find_type_full =
-  find (fun env -> env.types) (fun sc -> sc.comp_types)
 and find_modtype =
   find (fun env -> env.modtypes) (fun sc -> sc.comp_modtypes)
 and find_class =
   find (fun env -> env.classes) (fun sc -> sc.comp_classes)
 and find_cltype =
   find (fun env -> env.cltypes) (fun sc -> sc.comp_cltypes)
+and proto_find_type =
+  find (fun env -> env.types) (fun sc -> sc.comp_types)
+
+let normalize_path_ext p env =
```

Comment:
I don't understand why we need yet another normalization function that is different from all other normalization functions in the type-checker.